blocks/init/src/Blocks/custom/featured-posts/components/featured-posts-options.js

```diff
@@ -84,28 +153,70 @@ export const FeaturedPostsOptions = ({ attributes, setAttributes }) => {
 				<Spinner />
 			}
 
-			{(postTypeOptions[0] && posts) ?
+			{(postTypeOptions && taxonomyOptions) ?
 				<SelectControl
-					label={__('Posts Items', 'eightshift-frontend-libs')}
-					value={posts}
-					multiple
-					options={postsOptions}
+					label={sprintf(__('Filter by %s available taxonomies', 'eightshift-frontend-libs'), _.startCase(_.toLower(postType)))}
```

Comment:
It seems like `_.startCase(_.toLower(postType))` is used twice, you can extract this to a const and use here, no? Would be a bit more readable

Reply:
yea I could but that means extracting stuff upper in the component and calling it done here. If I like to put it here somewhere I would have to add additional Fragments around and that just complicates stuff. Also this value is dynamic so it depends on the data from the state. I would leave it like this.